streams/src/test/java/org/apache/kafka/streams/kstream/internals/SessionWindowedKStreamImplTest.java

```diff
@@ -132,28 +139,32 @@ public void apply(final Windowed<String> key, final String value) {
     public void shouldMaterializeCount() {
         stream.count(Materialized.<String, Long, SessionStore<Bytes, byte[]>>as("count-store"));
 
-        processData();
-        final SessionStore<String, Long> store = (SessionStore<String, Long>) driver.allStateStores().get("count-store");
-        final List<KeyValue<Windowed<String>, Long>> data = StreamsTestUtils.toList(store.fetch("1", "2"));
-        assertThat(data, equalTo(Arrays.asList(
-                KeyValue.pair(new Windowed<>("1", new SessionWindow(10, 15)), 2L),
-                KeyValue.pair(new Windowed<>("1", new SessionWindow(600, 600)), 1L),
-                KeyValue.pair(new Windowed<>("2", new SessionWindow(600, 600)), 1L))));
+        try (final TopologyTestDriver driver = new TopologyTestDriver(builder.build(), props, 0L)) {
+            processData(driver);
+            final SessionStore<String, Long> store = driver.getSessionStore("count-store");
+            final List<KeyValue<Windowed<String>, Long>> data = StreamsTestUtils.toList(store.fetch("1", "2"));
+            assertThat(data, equalTo(Arrays.asList(
+                    KeyValue.pair(new Windowed<>("1", new SessionWindow(10, 15)), 2L),
+                    KeyValue.pair(new Windowed<>("1", new SessionWindow(600, 600)), 1L),
+                    KeyValue.pair(new Windowed<>("2", new SessionWindow(600, 600)), 1L))));
+        }
     }
 
     @SuppressWarnings("unchecked")
     @Test
     public void shouldMaterializeReduced() {
         stream.reduce(MockReducer.STRING_ADDER, Materialized.<String, String, SessionStore<Bytes, byte[]>>as("reduced"));
 
-        processData();
-        final SessionStore<String, String> sessionStore = (SessionStore<String, String>) driver.allStateStores().get("reduced");
-        final List<KeyValue<Windowed<String>, String>> data = StreamsTestUtils.toList(sessionStore.fetch("1", "2"));
+        try (final TopologyTestDriver driver = new TopologyTestDriver(builder.build(), props, 0L)) {
```

Comment:
I think some of the above `@SuppressWarnings("unchecked")` can be removed now as well; could you double check?

Reply:
Yes, they can. Thank for catching this.